extensions/common/api/api-core/src/main/java/org/eclipse/edc/api/model/QuerySpecDto.java

```diff
@@ -40,6 +42,12 @@ public class QuerySpecDto {
     public static final String EDC_QUERY_SPEC_SORT_ORDER = EDC_NAMESPACE + "sortOrder";
     public static final String EDC_QUERY_SPEC_SORT_FIELD = EDC_NAMESPACE + "sortField";
 
+    @JsonProperty(value = CONTEXT)
+    private Object ldContext;
+
+    @JsonProperty(value = TYPE)
+    private String ldType;
+
```

Comment:
Is there a reason these are duplicated from the `BaseDto` class?

Reply:
no, I forgot them :sweat_smile: thanks